File: TOMIMSV4/Entity/Mantenimientos/Producto/Producto_Talla_Color/clsBeProducto_talla_color.vb
Public Class clsBeProducto_talla_color
	Implements ICloneable

	Public Property IdProductoTallaColor() As Integer = 0
	Public Property IdProducto() As Integer = 0
	Public Property IdTalla() As Integer = 0
	Public Property IdColor() As Integer = 0
	Public Property CodigoSKU() As String = ""
	Public Property IdCampaña() As Integer = 0
	Public Property Fec_agr() As Date = New Date(1900, 1, 1)
	Public Property User_agr() As String = ""
	Public Property Fec_mod() As Date = New Date(1900, 1, 1)
	Public Property User_mod() As String = ""
	Public Property Activo As Boolean = False

	Sub New()
		'#EJC: Add your constructor here... 
	End Sub

	Public Function Clone() As Object Implements System.ICloneable.Clone
		Return MyBase.MemberwiseClone()
	End Function

End Class

File: WMS.EntityCore/Producto/clsBeProducto_talla_color.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace WMSWebAPI.Be
{
    public class clsBeProducto_talla_color : ICloneable
    {
        [Column("IdProductoTallaColor")]
        [DisplayName("IdProductoTallaColor")]
        public int IdProductoTallaColor { get; set; } = 0;

        [Column("IdProducto")]
        [DisplayName("IdProducto")]
        public int IdProducto { get; set; } = 0;

        [Column("IdTalla")]
        [DisplayName("IdTalla")]
        public int IdTalla { get; set; } = 0;

        [Column("IdColor")]
        [DisplayName("IdColor")]
        public int IdColor { get; set; } = 0;

        [Column("IdCampaña")]
        [DisplayName("IdCampaña")]
        public int IdCampaña { get; set; } = 0;

        [Column("CodigoSKU")]
        [DisplayName("CodigoSKU")]
        public string CodigoSKU { get; set; } = "";

        [Column("fec_agr")]
        [DisplayName("fec_agr")]
        public DateTime Fec_agr { get; set; } = DateTime.Now;

        [Column("user_agr")]
        [DisplayName("user_agr")]
        public string User_agr { get; set; } = "";

        [Column("fec_mod")]
        [DisplayName("fec_mod")]
        public DateTime Fec_mod { get; set; } = DateTime.Now;

        [Column("user_mod")]
        [DisplayName("user_mod")]
        public string User_mod { get; set; } = "";

        public clsBeProducto_talla_color() { }

        public object Clone()
        {
            return this.MemberwiseClone();
        }
    }
}